compiler/lib/js_output.ml

```diff
@@ -886,9 +885,9 @@ struct
         let element f p =
           match p with
           | TargetElementHole -> ()
-          | TargetElementId (id, None) -> ident f id
+          | TargetElementId (id, None) -> ident f ~kind:`Binding id
           | TargetElementId (id, Some (e, _)) ->
-              ident f id;
+              ident f ~kind:`Binding id;
```

Comment:
Why is this a `Binding` while it is a `Reference` in TargetPropertyId ?

Reply:
Oh, this is wrong.